lib/ruby_lsp/executor.rb

```diff
@@ -26,6 +26,8 @@ def execute(request)
         response = run(request)
       rescue StandardError, LoadError => e
         warn(e.message)
+        backtrace = e.backtrace
+        warn(backtrace.join("\n")) if backtrace
```

Comment:
2 things here:

1. By using `Kernel#warn` we are making it so that `$VERBOSE = nil` or setting the verbosity level of the running Ruby interpreter will prevent these from going into the Output tab. I don't think that's what we want, so maybe we should explicitly start writing to `$stderr`.
2. I think `e.full_message` should output the message and the backtrace exactly as Ruby would have reported it: https://ruby-doc.org/3.0.3/Exception.html#full_message-method If we use that, then we don't need to worry about printing these things separately.